### PR TITLE
fix(nm): AP scan not reports correct Wifi Security

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR fixes the the Wifi Security mode reported by the AP scans.

**Related Issue:** This PR fixes/closes N/A

**Description of the solution adopted:** The current implementation of the modal for AP scan results doesn't report the correct Wifi Security mode. This PR fixes this wrong behavior in the web UI. The following security modes are supported:

- None
- WEP
- WPA
- WPA2
- WPA/WPA2
- WPA2/WPA3 Enterprise

To correctly determine the security, it is needed to retrieve a new property from NM. The `Flags` from the `AccessPoint` object is retrieved and shown also via the NetworkStatus Rest service.

**Any side note on the changes made:** Since WPA3 Personal/Enterprise is not supported, the scan filters the hotspots with this security mode.

### CODE REVIEW

File: kura/org.eclipse.kura.rest.network.status.provider/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -17,7 +17,7 @@ Import-Package: javax.annotation.security;version="1.2.0",
  org.eclipse.kura.net.status;version="[1.1,1.2)",
  org.eclipse.kura.net.status.ethernet;version="[1.0,1.1)",
  org.eclipse.kura.net.status.modem;version="[1.1,1.2)",
- org.eclipse.kura.net.status.wifi;version="[1.0,1.1)",
+ org.eclipse.kura.net.status.wifi;version="[1.1.0,1.2.0)",
```
Comment (MMaiero) [inline kura/org.eclipse.kura.rest.network.status.provider/META-INF/MANIFEST.MF]: [1.1,1.2)

File: kura/org.eclipse.kura.web2/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -70,7 +70,7 @@ Import-Package: com.eclipsesource.json;version="0.9.5",
  org.eclipse.kura.net.status.ethernet;version="[1.0,2.0)",
  org.eclipse.kura.net.status.modem;version="[1.1,2.0)",
  org.eclipse.kura.net.status.vlan;version="[1.0,2.0)",
- org.eclipse.kura.net.status.wifi;version="[1.0,2.0)",
+ org.eclipse.kura.net.status.wifi;version="[1.1.0,2.0.0)",
```
Comment (MMaiero) [inline kura/org.eclipse.kura.web2/META-INF/MANIFEST.MF]: [1.1,2.0)

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/status/NetworkStatusServiceAdapter.java [hunk 3]
```diff
@@ -159,21 +160,21 @@ public List<GwtWifiHotspotEntry> findWifiHotspots(String interfaceName, boolean
 
             for (WifiAccessPoint ap : wifiStatus.getAvailableWifiAccessPoints()) {
                 GwtWifiHotspotEntry entry = new GwtWifiHotspotEntry();
+                entry.setGroupCiphers(GwtWifiCiphers.netWifiCiphers_NONE.name());
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/status/NetworkStatusServiceAdapter.java:163]: Please update copyright header

File: kura/test/org.eclipse.kura.rest.network.status.provider.test/src/main/java/org/eclipse/kura/rest/network/status/provider/test/NetworkStatusRestServiceImplTest.java [hunk 4]
```diff
@@ -63,6 +63,7 @@
 import org.eclipse.kura.net.status.wifi.WifiAccessPoint;
 import org.eclipse.kura.net.status.wifi.WifiCapability;
 import org.eclipse.kura.net.status.wifi.WifiChannel;
+import org.eclipse.kura.net.status.wifi.WifiFlag;
```
Comment (nicolatimeus) [inline kura/test/org.eclipse.kura.rest.network.status.provider.test/src/main/java/org/eclipse/kura/rest/network/status/provider/test/NetworkStatusRestServiceImplTest.java:66]: Please update copyright header

**Review by nicolatimeus — APPROVED** [review]

**Pull request merged**